reesegreen2014/TaskPro
# Jarvis/feature/type checking and reflections

#### What does this PR do?
Adds a reflection page with detailed insights on integrating Redux with React for the TaskPro app.

#### Where should the reviewer start?
Review the `member1reflection` component

#### How should this be manually tested?
1. Navigate to the Jarvis reflection page.
2. Verify the content is displayed correctly with the appropriate styling.

#### Any background context you want to provide?
This reflection details the learning experience and benefits of using Redux for state management in the TaskPro app.

#### What are the relevant tickets (if any)?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
